### ISSUE TITLE
.chaise-dropdown .dropdown-menu>div>a introduces underlines in latest atlas-d2k.org navbar

### ISSUE THREAD

**Issue by crisaless** [informatics-isi-edu/chaise#2424]
The dev and staging servers for ATLAS are now displaying underlines under dropdown items in the navbar.

It looks like there's a new CSS handle? `.chaise-dropdown .dropdown-menu>div>a`

And it's pulling in the `text-decoration: underline` rule from the default bootstrap link.

Need to add `text-decoration: none`

**Comment by crisaless** [comment]
I also just noticed that if you add the new rule, the top-level menu items still shows text-decoration: underline on hover, which it shouldn't.

**Comment by RFSH** [comment]
The changes in https://github.com/informatics-isi-edu/chaise/commit/ebdb57105df79b7b8576b802bcc472301f575563 should solve this issue.

As part of https://github.com/informatics-isi-edu/chaise/pull/2413, I moved some navbar-specific rules to a common location (I needed to apply the same rules to all dropdowns and not just navbar). One of the rules that I moved was the following

```css
.dropdown .dropdown-toggle,
.dropdown a {
   text-decoration: none;
 }
```

But given that I moved it inside a `.chaise-dropdown` wrapper, the browser was using

```
.chaise-dropdown .dropdown .dropdown-toggle,
.chaise-dropdown .dropdown a {
   text-decoration: none;
 }
```

Which is invalid as `.dropdown` and `.chaise-dropdown` are attached to the same element. In my recent changes I changed the wrapper to be `.chaise-dropdown.dropdown` and simplified this rule to be only on `a` elements.
